### ISSUE TITLE
Add error handling on order/view when API 404 invoice-not-found

### ISSUE THREAD

**Issue by celi28** [btcpayserver/prestashop-plugin#35]
**Is your feature request related to a problem? Please describe.**
Due to a migration between BTCPay servers (with 1rst server lost, incl data), plugin can't fech the old invoices data.
Leading to a fatal error

```
An exception has been thrown during the rendering of a template ("Error during GET to https://btcpay.website.com/api/v1/stores/DHKa8TTypXFb9Xau9YU19aBiiFwFPYMMQXno6Lx13y6W/invoices/LAhsGNNLu7gFqFDvhp6S4E/payment-methods. Got response (404): {"code":"invoice-not-found","message":"The invoice was not found"}").
[Twig\Error\RuntimeError 0]

```

**Describe the solution you'd like**
Error handling, showing invoice as not found instead of raising uncaught 404 API call.

**Comment by BitcoinMitchell** [comment]
Sounds like a good idea, will implement this ASAP!

**Comment by BitcoinMitchell** [comment]
I've released [v5.1.2](https://github.com/btcpayserver/prestashop-plugin/releases/tag/v5.1.2) which should address this issue @celi28. Thank you for your report!
